### PR TITLE
Get application update

### PR DESCRIPTION
**Pull request by 00eadeyemi**
## Features
- 'Get an Application' documentation updated

[Delivers #111963201](https://www.pivotaltracker.com/story/show/111963201)

### CODE REVIEW

File: _includes/v4/applications_get.md [hunk 1]
```diff
@@ -25,12 +25,13 @@ Name | Required | Type | Description
 `$.finance.deposit_amount` | Yes | int | Deposit amount in pence that the customer has paid.
 `$.finance.subsidy_amount` | Yes | int | Amount of fees in pence that the application has attracted.
 `$.finance.settlement_net_amount` | Yes | int | The net amount that will be settled in pence.
+`$.finance.commission_amount` | Yes | int | The commission collected by the merchant in pence.
+`$.finance.option` | Yes | string | The code assigned to the loan product.
+`$.finance.option_group` | Yes | string | The code assigned to the loan product's group.
+`$.finance.holidays` | Yes | int | The number of monthly payment holidays given.
```
Comment (srleach) [inline _includes/v4/applications_get.md]: is field not `holiday` - and the description should be 'How many months until repayments begin.'

Comment (wnowicki) [inline _includes/v4/applications_get.md]:
```json
"finance": {
    "loan_amount": 29986,
    "order_amount": 29986,
    "deposit_amount": 0,
    "subsidy_amount": 2293,
    "settlement_net_amount": 27693,
    "commission_amount": 0,
    "option": "IFC-12",
    "option_group": "IFC",
    "holidays": 1,
    "payments": 12,
    "term": 13
  }
```

Don't guess ;)

But you are absolutely about description.

File: _includes/v4/applications_get.md [hunk 2]
```diff
@@ -25,12 +25,13 @@ Name | Required | Type | Description
...
+`$.finance.payments` | Yes | int | The number of monthly payments scheduled.
+`$.finance.term` | Yes | int |The total duration of the loan repayment period in months. 
 `$.metadata` | No | object | Metadata is used to add your own meaningful values to an application.
-`$.cancellation.requested` | No | bool | Has the cancellation of the application been requested?
```
Comment (wnowicki) [inline _includes/v4/applications_get.md]: why?

File: _includes/v4/applications_get.md [hunk 3]
```diff
@@ -43,8 +49,7 @@ Name | Required | Type | Description
         "last_name": "Labingi",
         "email_address": "fillibertlabingi+paybreak@gmail.com",
         "phone_home": null,
-        "phone_mobile": "07700900124",
-        "postcode": "TN12 6ZZ"
+        "phone_mobile": "07700900124"
```
Comment (EvanBarbour3) [inline _includes/v4/applications_get.md]: Postcode has been removed, so in the table above is it still using 'Applicant' object?

File: _includes/v4/applications_get.md [hunk 4]
```diff
@@ -66,18 +71,15 @@ Name | Required | Type | Description
         "group": "FF",
         "options": [
             "*"
-        ],
-        "default": "FF/1-3"
+        ]
```
Comment (EvanBarbour3) [inline _includes/v4/applications_get.md]: Should "default": "FF/1-3" have been removed?

**Pull request merged**